examples/uptime-monitor/steps/alerter.step.js

```diff
@@ -0,0 +1,86 @@
+const DISCORD_WEBHOOK =
+  "YOUR_WEBHOOK_URL"; // Add your Discord webhook URL here
```

Comment:
this should be an environment variable.

- create `.env.example` file with this content

```
DISCORD_WEBHOOK=http://your.webhook.url
```

- create `.env` file with the right variable
- put `.env` in `.gitignore`
- then just use `process.env.DISCORD_WEBHOOK`